lib/spritesheet.dart

```diff
@@ -28,6 +31,24 @@ class SpriteSheet {
                 height: textureHeight.toDouble())));
   }
 
+  SpriteSheet.fromImage({
+    @required Image image,
+    @required this.textureWidth,
+    @required this.textureHeight,
+    @required this.columns,
+    @required this.rows,
+  }) {
+    _sprites = List.generate(
+        rows,
+        (y) => List.generate(
```

Comment:
Multi-line anonymous functions should maybe use {} instead of the arrow notation? Or maybe define the `Sprite.fromImage` call as a local function and everything would fit.

Reply:
Done :D